### PR TITLE
feat: add option to display table actions even if table has no items

### PR DESCRIPTION
**Pull request by sshah-asymmetrik**
Hi John!

This PR is to add an option to the pageable-table.component that allows us to display table actions even if there are no items.

I tried to write it and set defaults so that any current users of the pageable-table won't see any differences.

Let me know if there are any additional changes I should make to get this merged.

Thanks!

### CODE REVIEW

File: src/app/common/paging/pageable-table/pageable-table.component.html [hunk 1]
```diff
@@ -61,13 +61,13 @@
 
 		<div class="table-footer-pager" *ngIf="pagerAtBottom" [ngClass]="{'ml-auto': !showFooterActions}">
```
Comment (jrassa) [inline src/app/common/paging/pageable-table/pageable-table.component.html]: Can we apply this to the footer actions as well?

File: src/app/common/paging/pageable-table/pageable-table.component.ts [hunk 2]
```diff
@@ -14,8 +14,9 @@ export class PageableTableComponent {
 	@Input() loading = false;
 	@Input() showInCard = false;
 	@Input() showActions = false;
+	@Input() hideActionsNoData = true;
```
Comment (jrassa) [inline src/app/common/paging/pageable-table/pageable-table.component.ts]: I think we should set the default to `false`.  That will make it's use consistent with `hideTableNoData`.  I think our 2 downstream projects are the only ones we have to worry about for compatibility and I am not actually using table actions anywhere currently.

**Review by jrassa — CHANGES_REQUESTED** [review]
This is a really good idea.  I actually avoided using the table actions in a few places because this option didn't exist.

**Comment by jrassa** [comment]
Also, if you wanted to, the admin pages could be updated to take advantage of this.

File: src/app/common/paging/pageable-table/pageable-table.component.html (hunk 1 above)
Comment (sshah-asymmetrik) [reply src/app/common/paging/pageable-table/pageable-table.component.html]: Good call! I added it as a separate option for footer actions, `hideFooterActionsNoData`.

File: src/app/common/paging/pageable-table/pageable-table.component.ts (hunk 2 above)
Comment (sshah-asymmetrik) [reply src/app/common/paging/pageable-table/pageable-table.component.ts]: Great! It is done.

**Review by sshah-asymmetrik — COMMENTED** [review]
I went ahead and shifted the actions in the admin view tables into tableActions.

**Pull request merged**